docs/WritingTests.asciidoc

```diff
@@ -97,14 +252,15 @@ sub test_flags {
 1;
 -------------------------------------------------------------------
 
-=== Test Case Examples
-[id="testcase_examples"]
+==== Install software via `zypper`
+[id="testmodule_perl_zypper"]
 
 [caption="Example: "]
 .Console test that installs software from remote repository via zypper command
+
 [source,perl]
 ----------------------------------------------------------------------------------------------------------
-sub run() {
+sub run {
```

Comment:
I don't mind bringing back the parentheses, I read the tests in distri opensuse and most of them didn't have the empty parentheses.

If there desired, I'll bring them back!

Reply:
Please keep the parentheses. And tests in os-autoinst-distri-opensuse are often not following any consistent pattern nor modern standards. The immense value of os-autoinst-distri-opensuse is in what tests it provides, not the code quality.

Reply:
Addressed in ebe9a6f8b

Reply:
Actually, the parentheses are superfluous.
A little background: Without activating the newer `signatures` feature, parentheses after subroutine names mean [prototypes](https://perldoc.perl.org/perlsub#Prototypes). An empty prototype means, this subroutine does not accept any argument.
However, object orientation was added to Perl later, and when calling a subroutine as a method (which we do for test modules), prototypes are ignored.
So I guess it's ok to keep the parentheses, but it would be more correct to remove them :)